src/main/java/com/jcraft/jsch/OpenSSHConfig.java

```diff
@@ -223,6 +241,24 @@ private String find(String key) {
         }
       }
       */
+
+      if (keysWithListAdoption.contains(key) && value != null && (value.startsWith("+") || value.startsWith("-") || value.startsWith("^"))) {
+
+        String origConfig = JSch.getConfig(originalKey).trim();
+
+        if (value.startsWith("+")) {
+          value=origConfig + "," + value.substring(1);
```

Comment:
Should we `trim()` the `value.substring(1)` here?

Reply:
yes